### ISSUE TITLE
cmd/go: new go tool "api" not documented in 1.12 release notes

### ISSUE THREAD

**Issue by ALTree** [golang/go#29836]
Go 1.12 exposes a new tool (`go tool api`):

```
$ diff -y -W 80 <(go tool) <(gotip tool)

addr2line				addr2line
				      >	api
asm					asm
buildid					buildid
cgo					cgo
compile					compile
cover					cover
dist					dist
doc					doc
fix					fix
link					link
nm					nm
objdump					objdump
pack					pack
pprof					pprof
test2json				test2json
tour				      <
trace					trace
vet					vet

```

But the newly exposed tool is not mentioned anywhere in the [1.12 release notes](https://tip.golang.org/doc/go1.12). I guess it should.

**Comment by ALTree** [comment]
cc @ianlancetaylor @andybons

**Comment by andybons** [comment]
Adding release-blocker so we don’t forget about this.

**Comment by ianlancetaylor** [comment]
I'm sure I'm missing something, but when I run `go tool` with Go 1.5 or later I see `api` in the list.  Is this a change in how the release is packaged?

**Comment by bradfitz** [comment]
We haven't shipped the api tool since 2015 (#13030).
